### PR TITLE
:tada: new planet from scratch

### PR DESCRIPTION
**Pull request by Boot-Error**
# 🎉 New Planet from scratch
Screenshot 2021-05-22 232754

## 🎇 Features
- Using a 11ty and tailwindcss to create a new planet from scratch
- Fixed known issues with relative links and incorrect link tags
- New layout, requires work on themes and adding additional links

### CODE REVIEW

**Comment by iayanpahwa** [comment]
Looks good.. 
cc : @xeon-zolt @insaaniManav

**Comment by xeon-zolt** [comment]
LGTM
I thought Animesh was also working on theme

**Comment by insaaniManav** [comment]
Bahut hi sahi  @Boot-Error , Merko to rss feed aggregator bhi chalta sirf bina UI ke ,  but this UI looks amazing

File: src/feeds/hellozee_blog.njk [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Hellozee's Blog"
```
Comment (hellozee) [inline src/feeds/hellozee_blog.njk:2]:
```suggestion
title: "hellozee's Blog"
```

**Comment by hellozee** [comment]
LGTM, ek chota change he bass, 😛

**Comment by iayanpahwa** [comment]
@Boot-Error the theme looks broke, please fix and re-open the PR, thanks!

**Comment by Boot-Error** [comment]
@iayanpahwa Fixed at #13

**Pull request merged**